### PR TITLE
Create Mock Data section

### PR DESCRIPTION
**Pull request by davemachado**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md).
- [ ] Your changes are made in the [README](../README.md) file, not the auto-generated JSON. 
- [ ] Your additions are ordered alphabetically.
- [ ] Your submission has a useful description.
- [ ] Each table column should be padded with one space on either side.
- [ ] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.
----------------------------------------------------------------------------------------------
Created Mock Data section to start pruning out our bloated `Development` section. Happy to hear other's thoughts, otherwise will merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -398,6 +388,21 @@ API | Description | Auth | HTTPS | Link |
...
+| Adorable Avatars | Generate random cartoon avatars | No | Yes | [Go!](http://avatars.adorable.io) |
+| Bacon Ipsum | A Meatier Lorem Ipsum Generator | No | Yes | [Go!](https://baconipsum.com/json-api/) |
+| Hipster Ipsum | Generates Hipster Ipsum text | No | No | [Go!](http://hipsterjesus.com/) |
+| JSONPlaceholder | Fake data for testing and prototyping | No | No | [Go!](http://jsonplaceholder.typicode.com/) |
+| Lorem Text | Generates Lorem Ipsum text | `X-Mashape-Key` | Yes | [Go!](https://market.mashape.com/montanaflynn/lorem-text-generator) |
+| Loripsum | The "lorem ipsum" generator that doesn't suck | No | No | [Go!](http://loripsum.net/) |
+| RandomUser | Generates random user data | No | Yes | [Go!](https://randomuser.me) |
+| RandomWord | Generate random word | No | No | [Go!](http://www.setgetgo.com/randomword/) |
+| RoboHash | Generate random robot/alien avatars | No | Yes | [Go!](https://robohash.org/) |
+| StackExchange | Q&A forum for developers | `OAuth` | Yes | [Go!](https://api.stackexchange.com/) |
```
Comment (jbrooksuk) [inline README.md]: Stack Exchange doesn't belong here.

**Comment by davemachado** [comment]
Oh no! I have moved Stack Exchange back to its real home 🏠

**Comment by mikestreety** [comment]
"Mock Data" doesn't seem right. Dummy Data? Test Data? 

And it's a fine line between an API and a placeholder image service? Are we including them? In which case I will add my very own: https://morganfillman.space/ 😉

**Comment by jbrooksuk** [comment]
I suggested "Testing" in the [issue](https://github.com/toddmotto/public-apis/issues/511#issuecomment-336558041). Test Data is probably better.

**Comment by davemachado** [comment]
@jbrooksuk sorry I see my reply never posted - I wasn't keen on Testing because these types of things don't really provide any testing logic...I'm good with Test Data though 😄

**Comment by jbrooksuk** [comment]
LGTM!

**Comment by davemachado** [comment]
🎉

**Pull request merged**